pages/datasets/_datasetId.vue

```diff
@@ -127,6 +127,11 @@
         :active-tab="activeTab"
         @set-active-tab="setActiveTab"
       >
+        <dataset-description-info
```

Comment:
This technically didn't need to be moved but is good for consistency.

Reply:
Yeah, just wanted to keep it consistent with the actual layout!